### ISSUE TITLE
Number of seconds per step

### ISSUE THREAD

**Issue by Zeta36** [ibab/tensorflow-wavenet#99]
Please, I have a simple question.

I'm training WaveNet with the VCTK corpus dataset in a CPU only machine (Intel i5) running to 20 sec/step. At this rate, I can reach about 4000 steps per day, 30.000 at week. Question is: ¿How much steps do we need to get a loss from about 2? And, another question: Can anybody tell me what is the seconds per step rate that a good GPU can reach? By the way, what's the rate in which each of you are working on guys?

Regards,
Samu.

Note. I'm using the default params:
{
    "filter_width": 2,
    "sample_rate": 16000,
    "dilations": [1, 2, 4, 8, 16, 32, 64, 128, 256,
                  1, 2, 4, 8, 16, 32, 64, 128, 256],
    "residual_channels": 32,
    "dilation_channels":16,
    "quantization_channels": 256,
    "skip_channels": 256,
    "use_biases": false
}

**Comment by r-zemblys** [comment]
Using default params I get loss of ~2 quite fast, in about 300 steps. You just need to lower learning rate, otherwise loss does not get lower that ~2.5.
I get 0.6-0.9 sec/step on TitanX GPU, using batch_size of 2

**Comment by Zeta36** [comment]
"I get 0.6-0.9 sec/step on TitanX GPU, using batch_size of 2"

Wow!!
I would love to have enough money to buy one of those.

Thank you for the info, @r-zemblys

**Comment by ibab** [comment]
I get about 0.5 sec / step on a K40 using `batch_size = 1`.

**Comment by Zeta36** [comment]
0.5 sec/step!! You can run more than 7k steps per hour!! I can run only  4k per day :P

Definitely, I have to buy a GPU for my computer.

**Comment by ibab** [comment]
@Zeta36: Right, there's an issue on it now: #112.

**Comment by aclaussen1** [comment]
How do you lower the learning rate?

**Comment by nakosung** [comment]
In my case 3 sec per step. Learning rate was annealed by scheduled shell
script.

2016년 12월 13일 (화) 오후 7:39, Alexander Claussen <notifications@github.com>님이
작성:

**Comment by yarodevuci** [comment]
I have 0.1 sec/step is that good?

**Comment by cgberlin** [comment]
yeah thats pretty good, i'm averaging 0.8 sec/step on a v100 using a custom dataset
